google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -349,6 +364,7 @@ private void recordAttemptCompletion(@Nullable Throwable status) {
             .put(CLIENT_NAME_KEY, NAME)
             .put(STREAMING_KEY, isStreaming)
             .put(STATUS_KEY, statusStr)
+            .put(TARGET_KEY, new ArrayList<>(this.targets))
```

Comment:
Im not sure you can have a list as a value for a metric attribute  value. Also what does it mean for a single attempt to have multiple targets?

Reply:
fixed